ingest/expression_files/mtx.py

```diff
@@ -116,6 +151,23 @@ def check_duplicates(names: List, file_type: str):
             raise ValueError(msg)
         return True
 
+    @staticmethod
+    def get_data_start_line_number(file_path: str) -> int:
+        """ Determines what line number data starts.
+
+         Returns
+         ----------
+            i (int) - Line number where data starts
+        """
+        i = 1
+        for line in file_path:
+            if line.startswith("%"):
+                i = +1
```

Comment:
This should be `i += 1`, otherwise `i` isn't being incremented.

Reply:
814b9c8